config/initializers/highlight_override.rb

```diff
@@ -59,3 +58,40 @@ def create_model!(user_uuid:)
     Highlight.create!(to_hash.merge(user_uuid: user_uuid))
   end
 end
+
+Api::V0::Bindings::Highlights.class_exec do
+  def self.create_from_models(highlights, pagination)
+    highlights = highlights.to_a unless highlights.is_a? Array
+    meta = pagination.merge(total_count: highlights.count)
```

Comment:
don't think this is what total count means

Reply:
i had thought total_count meant the number returned in the response.  i changed it to return the total count of all highlights involved in all pages.

Reply:
i fixed the total count to be everything.